engines/ags/detection_tables.h

```diff
@@ -346,13 +346,13 @@ const PlainGameDescriptor GAME_NAMES[] = {
 	{ "onironauta", "Onironauta" },
 
 	// AGDI games. They get their own grouping because they're just that awesome
-	{ "kq1agdi", "King's Quest I: Quest for the Crown Remake" },
+	{ "kq1agdi", "King's Quest: Quest for the Crown Remake" },
 	{ "kq2agdi", "King's Quest II: Romancing the Stones Remake" },
 	{ "kq3agdi", "King's Quest III Redux: To Heir is Human" },
 	{ "qfg2agdi", "Quest for Glory II: Trial by Fire Remake" },
 
 	// Infamous Adventures games. Likewise
-	{ "sq2fg", "Space Quest for Glory" },
+	{ "sq2fg", "Space Quest for Glory" }, //TODO Find proper Box Art for this and the following game to adjust the name
```

Comment:
I don't understand this TODO. There is no "official" boxart for these games, you can simply
refer to Infamous Adventures' itch.io page
https://infamousadventures.itch.io/

Reply:
When I was looking at them, I assume they are just space quest I and II but I could be wrong as well which is why I left that note, I couldn't decide which box art was released using the adventure game studio engine.

Reply:
@bluegr I went through and edited those. I am sorry I keep blundering the commits this is my first time working open source I appreciate the feed back.

Reply:
sq2fg is not a complete remake, it's a standalone game.
Like AGDI, Infamous Adventures releases are unofficial remakes, so there's no point using Sierra's original titles.

Reply:
@tag2015 Okay I'll go in and remove that comment thank you for the heads up